tests/test_utils.tsx

```diff
@@ -28,6 +28,23 @@ import { NodeProcess } from "./fixtures_islands/NodeProcess.tsx";
 import { FreshAttrs } from "./fixtures_islands/FreshAttrs.tsx";
 import { OptOutPartialLink } from "./fixtures_islands/OptOutPartialLink.tsx";
 
+/**
+ * If on Linux, disable `AppArmor` before running tests:
+ * ```sh
+ * echo 0 | sudo tee /proc/sys/kernel/apparmor_restrict_unprivileged_userns
```

Comment:
I'm not sure if it's a good idea to tell contributors on Linux to disable `AppArmor` globally on their machines. Contributing to a JS framework shouldn't come with the requirements to disable central security features in the OS you're using.

Reply:
Very good point. Silly for me to do. Reverted.